src/style.css

```diff
@@ -0,0 +1,84 @@
+/* No passed !!! */
+
+body {
+  margin: 0;
+  font-family: "Roboto", "sans-serif";
+}
+
+button {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;/* identical to box height */
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+  background: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  width: 100%;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  margin-top: 20px;
+}
+
+button:hover {
+  color: #00acdc;
+  background: #fff;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product {
+  padding: 30px 20px;
+  width: 158px;
+}
+
+.wrap {
+  padding: 11px 15px 15px;

```

Comment:
What does the `padding-top: 11px` do in this case? Better to set `margin-bottom: 45px` for the image wrapper. As in the mock.
![image](https://user-images.githubusercontent.com/29413892/56793417-aba71800-6814-11e9-946b-fc418531e37a.png)